website/docs/reference/database-permissions/postgres-permissions.md

```diff
@@ -2,8 +2,15 @@
 title: "Postgres Permissions"
 ---
 
+
+In Postgres, permissions are used to control who can perform certain actions on different database objects. Use SQL statements to manage permissions in a Databricks database.
+
 ## Example Postgres permissions
 
+The following example provides you with the SQL statements you can use to manage permissions. 
```

Comment:
From memory (@agnessnowplow correct me if I'm wrong) these ones were specifically chosen to allow the user to then run dbt without any permissions issues ( i.e. create schemas, read existing data, access the information schema). Could it be better to explain that here rather than list these as examples?

Reply:
@mirnawong1  Thank you for all the changes! Those permissions are definitely aimed at helping users run dbt without permission issues. I cannot not say it is a 100% full list but some of the most important ones for sure.